README.md

```diff
@@ -1,49 +1,48 @@
 # Local-links-manager
 
-Admin interface for managing Local Authorities links including all their services and interactions.
+Admin interface for managing Local Authorities links including all their
+services and interactions.
 
-This app maps RESTful URLs onto a persistence layer. It doesn't face public users.
-
-## Screenshots
-
-![Local Authorities](docs/images/local_authorities.png)
-
-![Local Authority Services](docs/images/services.png)
-
-![Local Authority Service Interactions](docs/images/interactions.png)
-
-![Interactions by Local Authority](docs/images/interactions_by_authority.png)
+This app maps RESTful URLs onto a persistence layer. It doesn't face public
+users.
 
 ## Nomenclature
 
-- **SNAC** - Standard Names And Code - The old identifier code for locations. This is being phased out in favour of GSS codes.
-- **GSS**  - Government Statistical Service - The new identifier code for locations.
+- **SNAC** - Standard Names And Code - The old identifier code for locations.
+  This is being phased out in favour of GSS codes.
+- **GSS**  - Government Statistical Service - The new identifier code for
+  locations.
 - **LGSL** - Local Government Services List
 - **LGIL** - Local Government Interactions List
 
-Both LGSL and LGIL codes are used for the lookups for each Local Authority and its service interactions.
+Both LGSL and LGIL codes are used for the lookups for each Local Authority and
+its service interactions.
 
-## Setup
+## Technical documentation
 
-The recommended way to run Local Links Manager is with [govuk-docker](https://github.com/alphagov/govuk-docker).
+This is a Ruby on Rails app, and should follow [our Rails app
+conventions][conventions].
 
-### Run with govuk-docker
+You can use the [GOV.UK Docker environment][govuk-docker] to run the
+application and its tests with all the necessary dependencies. Follow the
+[usage instructions][docker-usage] to get started.
 
-- Run `make local-links-manager` inside govuk-docker
-- Run `govuk-docker-run bundle exec rails db:seed` to create the default user
-- Run `govuk-docker-up` to start the application. Visit at http://local-links-manager.dev.gov.uk/.
-- `govuk-docker-run bundle exec rake` runs the tests.
+[conventions]: https://docs.publishing.service.gov.uk/manual/conventions-for-rails-applications.html
+[govuk-docker]: https://github.com/alphagov/govuk-docker
+[docker-usage]: https://github.com/alphagov/govuk-docker#usage
 
-### Run locally
+Running the bare application without any test data isn't very useful, you can
+replicate the app's Postgres database [using GOV.UK Docker][replicate-db].
 
-You will need to have [Mapit](https://github.com/alphagov/mapit) checked out locally,
-and you'll need to [import data from S3](https://github.com/alphagov/mapit/blob/master/import-db-from-s3.sh).
+[replicate-db]: https://github.com/alphagov/govuk-docker/blob/master/docs/how-tos.md#how-to-replicate-data-locally
```

Comment:
We've also been adding this at the end of the section:

```
**Use GOV.UK Docker to run any commands that follow.**
```

Reply:
Ah cool, I've added this to the section.